src/app/modules/user/user.middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { celebrate, Joi } from 'celebrate'
+
+/**
+ * Celebrate validation for the contact OTP verification endpoint.
+ */
+export const validateContactOtpVerificationParams = celebrate({
+  body: Joi.object({
```

Comment:
we normally use the `Segments` enum exposed by `celebrate` to avoid accidentally misnaming the keys. see `user.routes.ts` for reference